### PR TITLE
Added selected counties and venues to URLs so they can be bookmarked and emailed

### PR DESCRIPTION
**Pull request by mattjacobus**

### CODE REVIEW

**Comment by dsjoerg** [comment]
I really wanted to find bugs in Matt's code but haven't found any so far.

**Comment by mattjacobus** [comment]
:-)  I wish you would.  Need as many pointers as I can get.  Actually I just realized I left two console.logs in there, so those count!

File: static/ca_bydate.js [hunk 1]
```diff
@@ -229,6 +236,7 @@ function parseRow(row) {
 
 
 function parsingDone(results, file) {
+  console.log("parsingDone");
```
Comment (markroth8) [inline static/ca_bydate.js]: Remove debug statement

File: static/ca_bydate.js [hunk 2]
```diff
@@ -254,14 +262,13 @@ function parsingDone(results, file) {
...
+  populateSelect(countySel, counties, selectedCounties);
 
   locationTypeSel = document.getElementById('location-type-select');
-  populateSelect(locationTypeSel, locationTypes);
-
-  _.each([countySel, locationTypeSel], function(sel) { sel.addEventListener('change', redoFilter); });
+  populateSelect(locationTypeSel, locationTypes, selectedVenues);
 
   ageGroupSel = document.getElementById('agegroup-select');
+  console.log("ageGroupSel", ageGroupSel);
```
Comment (markroth8) [inline static/ca_bydate.js]: Remove debug statement

File: static/ca_bydate.js [hunk 3]
```diff
@@ -283,12 +290,28 @@ function parsingDone(results, file) {
...
     }
   });
+
+  redoFilter();
+
+  _.each([countySel, locationTypeSel], function(sel) {
+    sel.addEventListener('change', function() {
+      county = countySel.value ? countySel.value : ALL
+      venue = locationTypeSel.value ? locationTypeSel.value : ALL;
+      window.location = "/bydatesel/" + county + "/" + venue;
```
Comment (markroth8) [inline static/ca_bydate.js]: Use `encodeURIComponent()` for county and venue

**Comment by dsjoerg** [comment]
OK Matt I'll wait until the `console.log` statements are out and if you want to address @markroth8's comment and then I'll pull it in

File: static/ca_bydate.js (hunk 3 above)
Comment (mattjacobus) [reply static/ca_bydate.js]: I was doing this, but noticed the browser is automatically encoding these, so took it out.  Is this a browser compatibility issue?

**Comment by dsjoerg** [comment]
Also, there was a PR from Roberto here as well, so I merged his in, leaving you some conflicts to look at.  Hopefully not too hairy.

**Pull request merged**